.github/workflows/update-labels.yml

```diff
@@ -0,0 +1,22 @@
+name: Update label when user responds
+permissions:
+  issues: write
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  run-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: update labels when user responds
+        uses: andymckay/labeler@master
+        if: ${{ github.event.comment.user.login == github.event.issue.user.login && contains(github.event.issue.labels.*.name, 'wait for update') && !contains(github.event.issue.labels.*.name, 'user responded') }}
+        with:
+          add-labels: 'user responded'
+          remove-labels: 'wait for update'
+        if: ${{ github.event.comment.user.login == github.event.issue.user.login && contains(github.event.issue.labels.*.name, 'no response') && contains(github.event.issue.labels.*.name, 'wait for update') && !contains(github.event.issue.labels.*.name, 'user responded') }}
```

Comment:
Why do you add this condition `contains(github.event.issue.labels.*.name, 'no response') && contains(github.event.issue.labels.*.name, 'wait for update')`

Reply:
1st condition is for the following [question of yours](https://github.com/apache/apisix/pull/9927#discussion_r1282988073):
> What happends if the user respond an issue with the label "no response"?

2nd condition: We only care if the user responded when we are actually `waiting for response`.

Reply:
It's very werid, do you notice the logic relationship `&&`, does it mean the label should contain both `no response` and `wait for update`?

Reply:
OK, i get you